src/OutputWriters/windowed_time_average.jl

```diff
@@ -101,23 +101,27 @@ function next_actuation_time(sch::AveragedTimeInterval)
     N = sch.actuations
     T = sch.interval
     return t₀ + (N + 1) * T
+    # print("first_actuation_time:",t₀)
 end
-
-function (sch::AveragedTimeInterval)(model)
-    t = model.clock.time
-    t★ = next_actuation_time(sch)
-
-    if t >= t★
-        if sch.actuations < typemax(Int)
-            sch.actuations += 1
-        else # re-initialize the schedule to t★
-            initialize!(sch, t★)
-        end
-        return true
-    else
-        return false
-    end
+function actuations(sch::AveragedTimeInterval)
+    N = sch.actuations
+    return N
 end
```

Comment:
What is the purpose of this function? It doesn't seem to be used anywhere.

Reply:
yeah, there is no use for that, I have already deleted it I believe.